spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -214,7 +219,6 @@ def analyze(self):
         #     no angle correction is needed
         if self.fname_sc is not None:
             self.angle_correction()
-            self.get_midsagittal_slice()
```

Comment:
I notice that the call to `self.get_midsagittal_slice()` is removed here, but it isn't replaced with the equivalent call to `self.midsagittal_slice_interpolation()`. Instead, the call is placed inside the `measure` method, with a bunch of extra logic:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/4cfb3673da6e40e283f0a48992945c99ac1fe01e/spinalcordtoolbox/scripts/sct_analyze_lesion.py#L851-L862

Maybe what we could do is take the "get largest lesion" logic and put it inside of the midsagittal slice method, keep the old name `get_midsagittal_slice`, and put it in the same location it used to live at.

That way, it's very clear in the diff that all we're doing is updating the logic that we use to get the midsagittal slice, with the overall structure staying the same.

Reply:
excelent point. done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4702/commits/11cdd6567853945cd189a02a38da4c5b1838fc45